src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigDrawer.tsx

```diff
@@ -0,0 +1,191 @@
+            next: (resp) => {
+              if (resp && resp.ok) {
+                props.onCreate();
+              }
+            },
+            error: (err) => {},
+          })
+      );
+    },
+    [addSubscription, context.api, props.onCreate]
+  );
+
+  const onDefaultRecordingStart = React.useCallback(() => {
+    const config = context.settings.automatedAnalysisRecordingConfig();
+    const attributes = automatedAnalysisConfigToRecordingAttributes(config);
+    handleCreateRecording(attributes);
+  }, [context.settings, context.settings.automatedAnalysisRecordingConfig, handleCreateRecording]);
+
+  const onExpand = React.useCallback(() => {
+    drawerRef.current && drawerRef.current.focus();
+  }, []);
+
+  const onOptionSelect = React.useCallback(() => {
+    setIsDropdownOpen(false);
+    setIsExpanded(!isExpanded);
+  }, [setIsExpanded, setIsDropdownOpen, isExpanded]);
+
+  const onDrawerClose = React.useCallback(() => {
+    setIsExpanded(false);
+  }, [setIsExpanded]);
+
+  const panelContent = React.useMemo(() => {
+    return (
+      <DrawerPanelContent style={{ zIndex: 199 }}>
+        <DrawerHead>
+          <span tabIndex={isExpanded ? 0 : -1} ref={drawerRef}></span>
+          <DrawerActions>
+            <DrawerCloseButton onClick={onDrawerClose} />
+          </DrawerActions>
+        </DrawerHead>
+        <DrawerPanelBody>
+          <AutomatedAnalysisConfigForm onCreate={props.onCreate} isSettingsForm={false} />
+        </DrawerPanelBody>
+      </DrawerPanelContent>
+    );
+  }, [props.onCreate, onDrawerClose]);
+
+  const dropdownItems = [
+    <DropdownItem key="custom" onClick={onOptionSelect} icon={<CogIcon />}>
+      Custom
+    </DropdownItem>,
+    <DropdownItem key="default" onClick={onDefaultRecordingStart} icon={<PlusCircleIcon />}>
+      Default
+    </DropdownItem>,
+  ];
+
+  const dropdown = React.useMemo(() => {
+    return (
+      <Level>
+        <LevelItem style={{ margin: 'auto' }}>
+          <Dropdown
+            isFlipEnabled
+            menuAppendTo={'parent'}
+            toggle={
+              <DropdownToggle
+                aria-label="Recording Config Dropdown"
+                id="automated-analysis-recording-config-toggle"
+                splitButtonItems={[
+                  <DropdownToggleAction
+                    key="recording-cog-action"
+                    aria-label="Recording Actions"
+                    onClick={onOptionSelect}
+                  >
+                    <CogIcon />
+                  </DropdownToggleAction>,
+                ]}
+                onToggle={onToggle}
+                splitButtonVariant="action"
+                toggleVariant="default"
+              >
+                Create Recording &nbsp;&nbsp;
+              </DropdownToggle>
+            }
+            isOpen={isDropdownOpen}
+            dropdownItems={dropdownItems}
+          />
+        </LevelItem>
+      </Level>
+    );
+  }, [isDropdownOpen, onToggle, onOptionSelect]);
```

Comment:
missing deps on dropdownItems :D This won't be re-computed if dropDownItems change